tests/test_boundary_providers.py

```
@@ -216,3 +217,89 @@ def test_gadm_kwargs_forwarded(self):
         assert isinstance(provider, GADMProvider)
         assert provider._version == '3.6'
+
+
+# ---------------------------------------------------------------------------
+# RDHProvider
+# ---------------------------------------------------------------------------
+
+
+class TestRDHProvider:
+    def test_provider_name(self):
+        provider = RDHProvider(username='u', password='p')
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "password"

(S106)